bin/oneclick_library_import

```diff
@@ -1,6 +1,10 @@
 #!/usr/bin/env python
 """Update the circulation manager server with new books from the
-OneClick content server."""
+OneClick content server.
+
+If you want to test with sample OneClick data on prod database, run like so:
```

Comment:
That's fine, but I would take the word "prod" out of the comment. You definitely wouldn't want to run this with a production database that was actually in use.